chromadb/test/property/strategies.py

```diff
@@ -1,193 +1,318 @@
 import hypothesis
 import hypothesis.strategies as st
-from typing import Optional, TypedDict, Callable, List, cast
+from typing import Optional, TypedDict, Callable, List, Dict, Union, cast, TypeVar
 import hypothesis.extra.numpy as npst
 import numpy as np
 import chromadb.api.types as types
 import re
+from hypothesis.strategies._internal.strategies import SearchStrategy
+from hypothesis.strategies._internal.featureflags import FeatureStrategy
+from hypothesis.errors import InvalidArgument, InvalidDefinition
+
+from dataclasses import dataclass
 
 # Set the random seed for reproducibility
-np.random.seed(0)
+np.random.seed(0) # unnecessary, hypothesis does this for us
```

Comment:
If this is unnecessary why is it here?

Reply:
Does hypothesis override this also for how we currently generate embeddings, by making a random array?

Reply:
It's unclear if this means within hypothesis functions or globally, but think it's fine for our purposes for now.